docs/content/en/docs/installation-configuration/authentication.md

```diff
@@ -0,0 +1,253 @@
+{{< tab header="`.env`" lang="txt" >}}
+AUTH_ENABLED: true  
+OIDC_ISSUER: "https://authentikuri/application/o/does-providing-for-soarca/"
+OIDC_CLIENT_ID: "WxUcBMGZdI7c0e5oYp6mYdEd64acpXSuWKh8zBH5"
+OIDC_SKIP_TLS_VERIFY: true
+{{< /tab >}}
+
+{{< tab header="`bash`" lang="bash" >}}
+export AUTH_ENABLED=true
+export OIDC_ISSUER="https://authentikuri:9443/application/o/does-providing-for-soarca/"
+export OIDC_CLIENT_ID="WxUcBMGZdI7c0e5oYp6mYdEd64acpXSuWKh8zBH5"
+export OIDC_SKIP_TLS_VERIFY=true
+{{< /tab >}}
+{{< /tabpane >}}
+
+### Adding SOARCA user group and users
+
+{{% alert title="Note" color="primary" %}}
+Again, for the current version of the implementation we only support one group to differentiate between access to the different endpoint. We plan for a later version of SOARCA to have different groups/permissions for a given API endpoint.
+{{% /alert %}}
+
+Next, we require to setup a group in Authentik that is called `soarca_admin` as explained earlier. The to be obtained tokens from Authentik needs to have this group information as this will be checked by the middleware.
+
+![core](/SOARCA/images/installation_configuration/authentik_setup/groups.png)
+
+Under `users` normal as as service accounts can be created. We advise for machine-to-machine implementation service accounts, and for normal users (used for example for SOARCA-GUI logins) normal accounts. Now we can make an users and add to the `soarca_admin` group. Make use under the application that this group is added to the application provider that we have setup earlier, otherwise the grants of token might fail.
+
+![core](/SOARCA/images/installation_configuration/authentik_setup/add-user.png)
+
+![core](/SOARCA/images/installation_configuration/authentik_setup/add-group.png)
+
+### Authentication with Bearer
+
+Now that authentication and authorization is enabled, every request requires to have a set `Authorization: Bearer <token>` header.
+
+```
+POST /trigger/playbook/ HTTP/1.1
+Host: localhost:8080
+Content-Type: application/json
+Authorization: Bearer <token from authentik> 
+Content-Length: 2345
+
+{
+    "type": "playbook",
+    "spec_version": "cacao-2.0",
+    "id": "playbook--300270f9-0e64-42c8-93cc-0927edbe3ae7",
+    "name": "Example ssh",
+    ...
+}
+```
+
+The [gauth library](https://github.com/COSSAS/gauth) will validate this bearer token against the setup Authentik provider and grant the user or application access. Replace the token with a working bearer token.  
+
+{{% alert title="Tip" %}}
+For obtaining an access (bearer) token for Authentik, we have provided an example [here](https://github.com/COSSAS/SOARC/examples/docs/auth)
+{{% /alert %}}
+
+An example curl command is provided below:
+
+{{< tabpane langEqualsHeader=false  >}}
+{{< tab header="`bash`" lang="bash" >}}
+curl -X POST "http://localhost:8080/trigger/playbook/" \
+-H "Content-Type: application/json" \
+-H "Authorization: Bearer <replace token>" \
+-d '{
+    "type": "playbook",
```

Comment:
Just put the playbook in examples and use:

curl -X POST -H "Content-Type: application/json" -H "Authorization: Bearer <replace token>"  -d @./example/ssh-playbook.json localhost:8080/trigger/playbook